core/templates/pages/contributor-dashboard-page/opportunities-list/opportunities-list.component.html

```diff
@@ -14,18 +14,22 @@
     <oppia-opportunities-list-item></oppia-opportunities-list-item>
   </div>
   <div class="oppia-opportunity-pagination-container" *ngIf="visibleOpportunities.length > 0">
-    <button type="button"
-            class="oppia-pagination-button"
-            (click)="gotoPage(activePageNumber - 1)"
-            [disabled]="activePageNumber == 1">
-      <span class="material-icons">navigate_before</span>Previous
-    </button>
-    <button type="button"
-            class="oppia-pagination-button"
-            (click)="gotoPage(activePageNumber + 1)"
-            [disabled]="loadingOpportunityData || userIsOnLastPage">
-      Next<span class="material-icons">navigate_next</span>
-    </button>
+    <div class="button-wraper">
```

Comment:
oppia-pagination-button-wrapper

(also, note spelling of "wrapper")

Reply:
Thank you @seanlip ..I have fixed the typos and changed class name ..